osler/inventory/tests/factories.py

```diff
@@ -0,0 +1,50 @@
+import factory
+import datetime
+from factory.django import DjangoModelFactory
+from osler.inventory import models
+from osler.core.tests import factories as core_factories
+
+
+class DrugCategoryFactory(DjangoModelFactory):
+
+    class Meta:
+        model = models.DrugCategory
+
+    name = factory.Iterator(["Antibiotics", "Endocrine/GI", "Supplements", "Cardiovascular"])
+
+class MeasuringUnitFactory(DjangoModelFactory):
+
+    class Meta:
+        model = models.MeasuringUnit
+
+    name = factory.Iterator(["mg", "ml", "IU", "%"])
+
+class ManufacturerFactory(DjangoModelFactory):
+
+    class Meta:
+        model = models.Manufacturer
+
+    name = factory.Iterator(["Pfizer", "Amgen", "Regeneron", "Oxford BioMedica"])
+
+class DrugFactory(DjangoModelFactory):
+
+    class Meta:
+        model = models.Drug
+
+    name = 'Clindamycin'
+    unit = factory.SubFactory(MeasuringUnitFactory)
+    dose = 100.0
+    stock = 10
+    expiration_date = datetime.date(2020, 1, 1)
+    lot_number = 'ABCDEFGH'
```

Comment:
shouldn't this be a number?!

Reply:
Thanks for all the great suggestions btw, I'm not commenting on them from before, but they are very helpful!

Regarding this comment, the lot_number is a string of number and characters (caps) (and maybe even symbols(?), but it is a string).